EMS/client-helper-bundle/src/Helper/Search/Search.php

```diff
@@ -54,8 +58,13 @@ public function __construct(private readonly Request $request, ClientRequest $cl
             @\trigger_error('Deprecated facets, please use filters setting', E_USER_DEPRECATED);
         }
 
+        if (isset($options['fields']) && isset($options['query_search'])) {
```

Comment:
is $options['fields'] not always set? To []? 

```suggestion
        if (!empty($options['fields'] ?? []) && isset($options['query_search'])) {
```

Reply:
No this is just if it's set or not .. Or you set fields or you set query_search